route.scala

```diff
@@ -207,6 +208,21 @@ package object route {
             case applyType: Term.ApplyType =>
               Param(extractParamTerm(applyType, false, aliasDesc))
           }
+        case Term.ApplyType(Term.Name("params"), Seq(Type.Name(typeName))) =>
```

Comment:
stupid question ahead: wouldn't this be easier to read as quasi-quote?

```scala
case q"params[...$tpe]" => 
```

or something like it?